src/components/Nav.jsx

```diff
@@ -12,6 +11,17 @@ export const Nav = () => {
           <Link to="/articles">All Articles</Link>
         </li>
       </ul>
+      <ul className="topics-list">
+        <li>
+          <Link to="/coding">Coding</Link>
+        </li>
+        <li>
+          <Link to="/football">Football</Link>
+        </li>
+        <li>
+          <Link to="/cooking">Cooking</Link>
```

Comment:
Can we populate this list of topics using an API call rather than hardcoding? 🤔 
Currently if another category were ever added you'd have to edit this by hand, a herculean task